reference/docs-conceptual/learn/remoting/jea/role-capabilities.md

```diff
@@ -219,13 +219,13 @@ like you would with built-in and third-party modules.
 For tab completion to work properly in JEA sessions you must include the built-in function
 `tabexpansion2` in the **VisibleFunctions** list.
 
-## Place role capabilities in a module
+## Make the role capabilities available to a configuration
 
-In order for PowerShell to find a role capability file, it must be stored in a **RoleCapabilities**
-folder in a PowerShell module. The module can be stored in any folder included in the
-`$env:PSModulePath` environment variable, however you shouldn't place it in System32 or a folder
-where the untrusted, connecting users could modify the files. Below is an example of creating a
-basic PowerShell script module called **ContosoJEA** in the `$env:ProgramFiles` path.
+Prior to PowerShell 6, for PowerShell to find a role capability file it must be stored in a
+**RoleCapabilities** folder in a PowerShell module. The module can be stored in any folder included
+in the `$env:PSModulePath` environment variable, however you shouldn't place it in System32 or a
```

Comment:
Should System32 be more explicitly explained and maybe formatted?  Those who know what it is will know it's a folder in Windows, but those that don't may be unclear on where/what it is.

Reply:
Fixed this and tweaked some other wording.